packages/devextreme/scss/widgets/generic/list/_index.scss

```diff
@@ -118,6 +118,17 @@ $generic-list-search-editor-height: round($generic-base-line-height * $generic-b
 
     @include toggle-delete-icon($list-item-active-color);
   }
+
+  .dx-list-static-delete-button .dx-icon {
+    color: $list-icon-color;
+  }
+
+  &.dx-state-active,
+  &.dx-state-focused {
+    .dx-list-static-delete-button .dx-icon {
```

Comment:
The problem can also arise not only for the delete button but also for any other buttons that may be added by the user using a template.